.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
```

Comment:
``.composer/cache/files`` would be enough, to avoid invalidating the cache because of Packagist metadata